app/helpers/effective_truncate_html_helper.rb

```diff
@@ -28,9 +28,8 @@ def truncate_html(text, max = 200, read_more = '', ellipsis = '...')
           textNode = element.children.first
           textNode.content = truncate(textNode.content, length: (max - length), separator: ' ', omission: ellipsis)
           break # Break out of our while loop, as our ellipsis might invalidate our looping condition
-        else # Unexpected, so just remove the whole thing
-          Rails.logger.info "effective_posts, Unexpected number of last_element_child children"
-          element.remove
```

Comment:
I'd prefer to attempt to truncate the text nodes instead of just remove them..

I'll bug you about where on SE you ran into this bug tuesday.  I went through every post and did not encounter an infinite loop